info-compiler/src/test/java/com/google/sps/NewsContentExtractorTest.java

```diff
@@ -65,15 +65,15 @@
   private NewsContentExtractor realNewsContentExtractor = new NewsContentExtractor();
```

Comment:
Reminder to remove any of these constants that are now obsolete.

Reply:
realNewsContentExtractor is used in this [test](https://github.com/googleinterns/voter-central/blob/5f23fd69571b9fb59ae42fb01f3d469ca032bf99/info-compiler/src/test/java/com/google/sps/NewsContentExtractorTest.java#L149) and the following one. Those two tests are meant to test if ContentExtractor can catch exceptions or null, so the parser is the real parser that reads a bad stream input

Reply:
The real parser doesn't read from any real webpages, it just calls on InputStream to read something, which will trigger an exception that is meant to be caught.

Reply:
*I actually meant NEWS_ARTICLE_WITH_EMPTY_TITLE_CONTENT, etc. but couldn't comment up there since there wasn't a recent  change (sorry!)

Reply:
Oh gotcha! I missed those. Those are fixed in the commit just now.
Thank you!